web/packages/teleport/src/Integrations/Enroll/AwsConsole/Access/Access.tsx

```diff
@@ -47,64 +54,119 @@ export function Access() {
   const ctx = useTeleport();
   const integrationsAccess = ctx.storeUser.getIntegrationsAccess();
   const canEnroll = integrationsAccess.create;
-  const clusterId = ctx.storeUser.getClusterId();
-  const resourceRoute = cfg.getUnifiedResourcesRoute(clusterId);
 
   const history = useHistory();
   const location = useLocation<{
     integrationName?: string;
     trustAnchorArn?: string;
-    syncRoleArn?: string;
     syncProfileArn?: string;
+    syncRoleArn?: string;
+    edit?: boolean;
   }>();
 
   const {
     integrationName = '',
     trustAnchorArn = '',
-    syncRoleArn = '',
     syncProfileArn = '',
+    syncRoleArn = '',
+    edit = false,
   } = location.state;
-  const [syncAll, setSyncAll] = useState(true);
+  const [syncAll, setSyncAll] = useState(!edit);
   const [filters, setFilters] = useState<ProfilesFilterOption[]>([]);
+  // initialFilters is used in the edit flow to reset filters if import all is un-toggled
+  const [initialFilters, setInitialFilters] = useState<ProfilesFilterOption[]>(
+    []
+  );
+  // initialComplete & foundProfiles allows us to show a custom view for zero profiles,
+  // vs. showing the table view when the user has filtered down to zero
+  const initialComplete = useRef(false);
+  const [foundProfiles, setFoundProfiles] = useState(false);
 
-  // todo (michellescripts) list profiles as written today fails
-  //   meeting with marcoandredinis next week to correct
-  const { status, error, data, isFetching, refetch } = useQuery({
-    enabled: canEnroll,
-    queryKey: ['profiles', filters],
-    gcTime: 0,
-    queryFn: () =>
-      integrationService.awsRolesAnywhereProfiles({
-        integrationName: integrationName,
-        filters,
-      }),
-    placeholderData: keepPreviousData,
-    staleTime: 30_000, // Cached pages are valid for 30 seconds
+  const [intervalMs, setIntervalMs] = useState<number | false>(2000);
```

Comment:
should we set to `false` if in edit mode?

Reply:
Added a story too.